### DISCUSSION TITLE
URGENT: Return each word probability

### DISCUSSION THREAD

**Discussion by hungphongtrn** [openai/whisper#330] [Q&A] [answered]
Currently, instead of the cumulative probability of the sequence, I need to return a list of probabilities for each word of the sequence. I am attending a Hackathon related to ASR, therefore I would love to receive support ASAP. 

Thank you all for spending time.

**Comment by Arlen22** [comment]
https://github.com/openai/whisper/blob/d18e9ea5dd2ca57c697e8e55f9e654f06ede25d0/whisper/decoding.py#L643

It looks like this is where you would get that result. I don't know for sure, I just tried to trace the code flow.

**Comment by jianfch** [comment, marked as answer]
check #284
